tests/components/voip/test_voip.py

```diff
@@ -1120,54 +1145,21 @@ async def test_start_conversation_user_doesnt_pick_up(
     mock_protocol: AsyncMock = hass.data[DOMAIN].protocol
     mock_protocol.outgoing_call = Mock()
 
-    pipeline_started = asyncio.Event()
-
-    async def async_pipeline_from_audio_stream(
-        hass: HomeAssistant,
-        context: Context,
-        *args,
-        conversation_extra_system_prompt: str | None = None,
-        **kwargs,
-    ):
-        # System prompt should be not be set due to timeout (user not picking up)
-        assert conversation_extra_system_prompt is None
-
-        pipeline_started.set()
+    announcement = assist_satellite.AssistSatelliteAnnouncement(
+        message="test announcement",
+        media_id=_MEDIA_ID,
+        original_media_id=_MEDIA_ID,
+        media_id_source="tts",
+    )
 
+    # Very short timeout which will trigger because we don't send any audio in
     with (
         patch(
-            "homeassistant.components.assist_satellite.entity.async_get_pipeline",
-            return_value=pipeline,
-        ),
-        patch(
-            "homeassistant.components.voip.assist_satellite.VoipAssistSatellite.async_start_conversation",
-            side_effect=TimeoutError,
-        ),
-        patch(
-            "homeassistant.components.assist_satellite.entity.async_pipeline_from_audio_stream",
-            new=async_pipeline_from_audio_stream,
-        ),
-        patch(
-            "homeassistant.components.assist_satellite.entity.tts_generate_media_source_id",
-            return_value="test media id",
+            "homeassistant.components.voip.assist_satellite._ANNOUNCEMENT_RING_TIMEOUT",
+            0.1,
         ),
     ):
-        satellite.transport = Mock()
+        satellite.connection_made(Mock())
 
-        # Error should clear system prompt
```

Comment:
Since the system prompt logic is just in the assis_satellite component I thought it was reasonable to remove those checks from here. It should be sufficient to just test that the error is raised here.